src/VisualStudio/Core/Def/Telemetry/VSTelemetryActivityLogger.cs

```diff
@@ -17,7 +17,6 @@ internal sealed class VSTelemetryActivityLogger : ForegroundThreadAffinitizedObj
         private static readonly HashSet<FunctionId> s_functionIds
             = new HashSet<FunctionId>()
             {
-                FunctionId.NavigateTo_Search,
```

Comment:
I feel like i've spent longer talking about this then it would have taken you to fix this :)